index.html

```diff
@@ -21,8 +22,8 @@ <h2>Question</h2>
           impedit debitis odio asperiores iusto aliquid ipsam cum facere amet
           aliquam illum quidem et!
         </p>
-        <button class="button">Button text</button>
-        <p class="answer">
+        <button data-js="button1" class="button">Button text</button>
```

Comment:
I would suggest to change "button1" to "button" - without the number at the end. For all buttons.